pkg/endpoint/policy.go

```diff
@@ -255,6 +255,7 @@ func (e *Endpoint) regeneratePolicy() (retErr error) {
 		e.getLogger().Debug("Forced policy recalculation")
 	}
 
+	repo.UpdatePolicyMetrics(e.SecurityIdentity, true)
```

Comment:
This call should not be done on the endpoint level, would be better to do it within `policy.resolvePolicyLocked()` instead.